packages/common/src/ui/components/inputs/Switch/Switch.tsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
```

Comment:
styled the switch to look like the version in zeplin. Not that keen on the square-ish border, that's personal preference I guess.

Reply:
The 'off' state I took from [the requisition filter design](https://app.zeplin.io/project/60e44fff6fa942168092f1ff/screen/619a9523bb78cca911bba916) in zeplin which is:

![Screen Shot 2021-12-14 at 3 05 01 PM](https://user-images.githubusercontent.com/9192912/145919667-2b6e7f81-ab5e-4e8f-906d-49867d863617.png)

I see now, that there's a more rounded version hiding in the background!

![Screen Shot 2021-12-14 at 3 06 15 PM](https://user-images.githubusercontent.com/9192912/145919770-84d452c2-6a00-4bdb-a63c-5f37181e8993.png)
 
Just no 'on' state.

Reply:
🤣  I thought this was the 'on' state 😁